### DISCUSSION TITLE
cron permissions

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#17067] [General]
Hello,

When I add a new user, it takes a long time for cron to update the accesses, more than 24 hours. During this period, the new records added by other users are not visible in the modules.
![ConfReport](https://user-images.githubusercontent.com/75428743/236665481-f3cdd77d-dccf-4e60-baba-2b48d08603b1.png)

**Comment by rskrzypczak** [comment]
in general you have 3 critical configuration errors here:
- uopz (must be removed/inactive)
- app_data/cron.php
- public_html (highly recommended for security reasons)

Looks like you have a problem with cron.
FYI: https://doc.yetiforce.com/administrator-guides/automation/cron/
